.vib/cassandra/goss/goss.yaml

```diff
@@ -0,0 +1,31 @@
+file:
+  {{ .Vars.persistence.mountPath }}:
+    exists: true
+    filetype: directory
+    mode: "2775"
+    owner: root
+  /opt/bitnami/cassandra/conf/cassandra.yaml:
+    exists: true
+    filetype: file
+    mode: "0644"
+    group: root
+    contains:
+      - /num_tokens.*{{ .Vars.cluster.numTokens }}/
+  /var/run/secrets/kubernetes.io/serviceaccount:
+    exists: {{ .Vars.serviceAccount.create }}
+    filetype: directory
+    mode: "3777"
+command:
+  {{- $user := .Vars.dbUser.user }}
+  {{- $password := .Vars.dbUser.password }}
+  {{- $nodes := .Vars.cluster.seedCount }}
```

Comment:
I think the number of nodes != to the number of seed nodes. IIRC, the former is determined by `replicaCount`.

Reply:
It is for this testing environment where I'm setting both in the `runtime_params` but I understand what you mean. Using `replicaCount` now.